CryptoPkg/Library/BaseCryptLibMbedTls/Pk/CryptEc.c

```diff
@@ -405,20 +405,25 @@ EcPointAdd (
 {
   mbedtls_mpi  *m;
   mbedtls_mpi  *n;
+  BOOLEAN      Status;

```

Comment:
Use mbedtls_mpi  m will be better

Reply:
Thanks. I have deleted the malloc operation.